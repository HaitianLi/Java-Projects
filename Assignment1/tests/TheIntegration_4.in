add book books.csv 111111
add collection books.csv
ADD CollecTioN books2.csv
list all
add member jack hu
add member Mr ma
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
relinquish all 100000
rent 100001 111111
rent 100001 111112
rent 100001 111113
list available
relinquish all 100001
list available
member history 100001
common 100000 100001
exit


