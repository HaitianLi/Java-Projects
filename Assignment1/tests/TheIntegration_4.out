user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: 16 books successfully added.

user: 19 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)
should come first 111144 (Haitian Li)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
final with 211204 (Bob Kane)
third show with 211203 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
first show with 211120 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)
should come first 111144 (Haitian Li)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
final with 211204 (Bob Kane)
third show with 211203 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
first show with 211120 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)
should come first 111144 (Haitian Li)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
final with 211204 (Bob Kane)
third show with 211203 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
first show with 211120 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Ending Library process.
